src/cbmc/cbmc_parse_options.cpp

```diff
@@ -1041,6 +1041,9 @@ void cbmc_parse_optionst::help()
     "\n"
     "Backend options:\n"
     " --object-bits n              number of bits used for object addresses\n"
+    // NOLINTNEXTLINE(whitespace/line_length)
+    " --malloc-fail-assert         set malloc failure mode to assert-then-assume\n"
```

Comment:
I think these should really be in `Program instrumentation options` because they are controlling what properties are checked.

Reply:
Agree with Martin here too.